administration/src/bp-modules/cards/hooks/useCardGenerator.ts

```diff
@@ -59,6 +66,23 @@ const useCardGenerator = ({ region, initializeCards = true }: UseCardGeneratorPr
   const [createCardsMutation] = useCreateCardsMutation()
   const [deleteCardsMutation] = useDeleteCardsMutation()
   const appToaster = useAppToaster()
+  const { t } = useTranslation('cards')
+
+  const [sendToFreinet] = useSendApplicationDataToFreinetMutation({
+    onCompleted: data => {
+      if (data.sendApplicationDataToFreinet === true) {
+        appToaster?.show({ intent: 'success', message: t('freinetDataSyncSuccessMessage') })
+      } else {
+        appToaster?.show({ intent: 'danger', message: t('errors:freinetDataSyncError') })
+      }
+    },
+    onError: error => {
+      if (error.message.includes('FREINET_DATA_TRANSFER_NOT_ACTIVATED') === false) {
```

Comment:
🔧 I wouldn't throw an error if its not activated in the backend. Then this check is not needed